discord/ui/view.py

```diff
@@ -465,6 +465,19 @@ async def wait(self) -> bool:
         """
         return await self.__stopped
 
+    def disable_all_items(self, *, exclusions: Optional[List[Item]]) -> None:
+        """
+        Disables all items in the view.
+
+        Parameters
+        ----------- 
+        exclusions: Optional[List[:class:`ui.Item`]]
+            A list of items in `self.children` to not disable from the view.
+        """
+        for child in self.children:
+            for i in exclusions:
+                child.disabled = True if child != i else None
```

Comment:
Logic here doesn't really make sense, would change it to this
```suggestion
            if child not in exclusions:
                child.disabled = True
```